start-idsvr.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+####################################################################################################
+# A developer deployment to run the Curity Identity Server in Docker on the local computer.
+# The deployment provides a working configuration and includes native passkeys automation.
+# - https://curity.io/resources/learn/mobile-logins-using-native-passkeys/
+####################################################################################################
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+#
+# First check for a license
+#
+if [ ! -f './license.json' ]; then
+  echo 'Please copy a license.json file into the root folder of the code example'
+  exit 1
+fi
+
+#
+# Then validate or default parameters
+#
+EXAMPLE_NAME='haapi'
+if [ "$USE_NGROK" != 'true' ]; then
+  USE_NGROK='false'
+  if [ "$IDSVR_HOST_NAME" == '' ]; then
+    echo 'You must supply an IDSVR_HOST_NAME for the Curity Identity Server'
+    exit 1
+  fi
+fi
+
+#
+# Download mobile deployment resources
+#
+rm -rf deployment
```

Comment:
Maybe this could be wrapped in a check if the deployment scripts already exist? I know that this will require manually updating the deployment scripts when they change, but will limit the amount of cloning we make. It will also allow making manual changes to the deployment scripts. The same applies to the `stop-idsvr` script.

Reply:
Sounds good